lib/Request.js

```diff
@@ -38,6 +38,27 @@ class Request {
   }
 
   /**
+   * Rename the key (if rename function exists) when used in map or filter.
+   *
+   * @param {String} key The string to rename
+   * @return {String} key The resulting string
+   */
+
+  rename(key) {
+    // Rename if rename function exists
+    if (this.target.rename) {
+      const prefixIndexBeg = key.indexOf(this.target.prefix)
```

Comment:
`key` is the output key here, so `key.indexOf(this.target.prefix)` will always be equal to `0`